src/logic/export/PolygonLabelsExporter.ts

```diff
@@ -62,19 +63,22 @@ export class PolygonLabelsExporter {
         if (!validLabels.length) return null;
 
         return validLabels.reduce((data: VGGRegionsData, label: LabelPolygon, index: number) => {
-            data[`${index}`] = {
-                shape_attributes: PolygonLabelsExporter.mapPolygonToVGG(label.vertices),
-                region_attributes: {
-                    label: labelNames[label.labelIndex]
-                }
-            };
+            const labelName: LabelName = _.findLast(labelNames, {id: label.labelId});
+            if (!!labelName) {
+                data[`${index}`] = {
```

Comment:
skoro `index` jest typu `number`, to dlaczego robisz to w taki sposób z backtickami?